less/datepicker3.less

```diff
@@ -50,26 +50,21 @@
 			border-top:    6px solid #fff;
 		}
 	}
-	>div {
+	> div {
 		display: none;
 	}
-	&.days div.datepicker-days {
+	&.days .datepicker-days,
+	&.months .datepicker-months,
+	&.years .datepicker-years {
 		display: block;
 	}
-	&.months div.datepicker-months {
-		display: block;
-	}
-	&.years div.datepicker-years {
-		display: block;
-	}
-	table{
+	table {
 		margin: 0;
 		-webkit-touch-callout: none;
 		-webkit-user-select: none;
 		-khtml-user-select: none;
 		-moz-user-select: none;
 		-ms-user-select: none;
-		user-select: none;
```

Comment:
Maybe we should keep the property as it does no harm to the code. Csslint also throws a warning for it when the `user-select`property is not set.

```
WARNING: Missing standard property 'user-select' to go along with '-moz-user-select'. When using a vendor-prefixed property, make sure to include the standard one. (vendor-prefix) Browsers: All
```

so my idea is to keep the property